swift/src/Ice/CommunicatorI.swift

```diff
@@ -39,6 +39,15 @@ class CommunicatorI: LocalObject<ICECommunicator>, Communicator {
         handle.waitForShutdown()
     }
 
+    func shutdownCompleted() async {
+        async let task = Task {
```

Comment:
I verified this code works as expected. Can we write it in a simpler fashion?

Reply:
I believe you can just do `await Task { ... }.value`. No need for the async let.